src/index.ejs

```diff
@@ -1,7 +1,6 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta http-equiv="Content-Security-Policy-Report-Only" content="default-src 'self'; font-src data: 'self'; connect-src 'self' api.amplitude.com 9sl3dhr1twv1.statuspage.io; report-uri https://sentry.io/api/1773525/security/?sentry_key=478778eb1a5642de97c08f3c3ea33ece;"></meta>
```

Comment:
This is applied via Ops configuration as a header, and this was logging a warning due to `report-uri` in a <meta> tag. Nuked.